misc/python/materialize/checks/owners.py

```diff
@@ -87,7 +114,11 @@ def validate(self) -> Testdrive:
         )
         # TODO: Fix owners in dbs, schemas, types after #18414 is fixed
         return Testdrive(
-            self._create_objects("owner_role_01", 5)
+            self._drop_objects("materialize", 1, success=False, expensive=True)
```

Comment:
Can you put a comment here to the effect that role "materialize" is not allowed to drop those objects.